README.md

```diff
@@ -4,63 +4,75 @@
 [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
 [![Stylelint Style Guide](https://img.shields.io/badge/code_style-stylelint-brightgreen.svg)](https://github.com/alphagov/stylelint-config-gds/)
 
-A Ruby gem to document and distribute components for GOV.UK applications. It provides:
+govuk_publishing_components is a Ruby gem to document and distribute components for GOV.UK applications. It provides:
 
 - a shared library of components
 - helpers to generate component payloads
 - an application to preview components and provide guidance
 - [GOV.UK Design System](https://design-system.service.gov.uk/) styles and components (via [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend))
 
-Components should be added to this gem if they are required in more than one application, otherwise they should be added to that application. All components should meet the same standards of browser and assistive technology support described in [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend#browser-and-assistive-technology-support).
+Only add a component to this gem if it is required in more than one application. Please do not add application-specific components to govuk_publishing_components.
+
+All components should meet the accessibility standards for browser and assistive technology support described in [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend#browser-and-assistive-technology-support).
 
 ## Technical documentation
 
-You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) or the local `startup.sh` script to run the app. Read the [guidance on local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html) to find out more about each approach, before you get started.
+### Run the application
+
+```sh
+./startup.sh
+```
 
-If you are using GOV.UK Docker, remember to combine it with the commands that follow. See the [GOV.UK Docker usage instructions](https://github.com/alphagov/govuk-docker#usage) for examples.
+See also [Local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html).
 
-### Running the test suite
+### Run the test suite
 
 ```sh
+bundle install
+yarn install
 bundle exec rake
 ```
 
-To run JavaScript tests (only):
+### Run the JavaScript tests alone
+
+In browser:
 
 ```sh
-# run JS tests in browser
 yarn run jasmine:browser
+```
```

Comment:
Just realised this doesn't state that the tests can be seen in a browser at http://localhost:8888 - do you think it's worth adding? (non-blocking)

Reply:
Also worth noting that the test server needs a restart to pick up any changes to the JavaScript - again potentially out of scope, might be better to link to another page to go more in depth. Happy to raise a separate PR if you think it's useful.

Reply:
Ah cool, thanks! I'll add.

Reply:
Done!